db/migrate/20160516070529_divide_lookup_key_permissions.rb

```diff
@@ -1,14 +1,32 @@
 class DivideLookupKeyPermissions < ActiveRecord::Migration
+  class FakeFilter < ActiveRecord::Base
```

Comment:
I'm trying to find a way to fix the migration that [fails](http://ci.theforeman.org/job/test_branch_upgrade/11214/database=postgresql,ruby=2.0.0/console) because of the new Filter validation. The same old problem, people are using models in migrations when they create/manipulate data. The problem is that it can later break because the migration uses latest version of model that can e.g. have a validation on field that does not exist in DB when this migration is run. We should never merge a PR that adds migration that uses model to create or update the DB in a way that triggers AR callbacks.